cibuild.cmd

```diff
@@ -58,14 +59,19 @@ if /i "%HOST%"=="CoreCLR" (
     goto :error
 )
 
+set LOCALIZED_BUILD_ARGUMENT=
+if "%LOCALIZED_BUILD%"=="true" (
+    set LOCALIZED_BUILD_ARGUMENT="/p:LocalizedBuild=true"
+)
+
 :: Restore build tools
 call %~dp0init-tools.cmd
 
 echo.
 echo ** Rebuilding MSBuild with downloaded binaries
 
 set MSBUILDLOGPATH=%~dp0msbuild_bootstrap_build.log
-call "%~dp0build.cmd" /t:Rebuild /p:Configuration=%BUILD_CONFIGURATION% /p:"SkipBuildPackages=true"
+call "%~dp0build.cmd" /t:Rebuild /p:Configuration=%BUILD_CONFIGURATION% /p:"SkipBuildPackages=true" %LOCALIZED_BUILD_ARGUMENT%
```

Comment:
Does the first bootstrapping build need to be localized? Seems like it's a final-build-only kind of thing.

Of course, we don't have good logic for the `--bootstrap-only` case.